bin/oref0-ns-autotune.sh

```diff
@@ -112,6 +119,12 @@ cp settings/autotune.json autotune/profile.json && cat autotune/profile.json | j
 cd autotune
 # TODO: Need to think through what to remove in the autotune folder...
 
+# Turn on stdout logging, if enabled (default to true)
```

Comment:
I had assumed that stdout was where all of the messaging was coming from in the prep/autotune process. All that output is what's in the log files. We can change the terminology here if needed, no problem.